### ISSUE TITLE
TypeError: visit_Constant() got an unexpected keyword argument 'containingNodes' still present

### ISSUE THREAD

**Issue by sWahlinRhoades** [Feneric/doxypypy#85]
Hi, I just installed the new version of doxypypy using pip (0.8.8.6) And I am still getting the error reported when I attempt to run doxypypy using python 3.8.12,

I noticed in issue #70 this is marked as resolved. Has this resolved release just not made it out of the repository yet?
 @Feneric 

```
TypeError: visit_Constant() got an unexpected keyword argument 'containingNodes'
```

**Comment by Feneric** [comment]
Yes, what's checked in to the repository is ahead of the official release. There are a few more things that'll hopefully get fixed before the next release is cut.

**Comment by suurjaak** [comment]
Any chance of getting an official PyPi release any time soon? The current version there is 7 years old already, and this error in newer Pythons makes doxypypy very difficult to use.

**Comment by gabrielreale** [comment]
I am also experiencing this issue. Waiting for the new PyPi release to try it as it seems to be a very useful tool.

**Comment by grbd** [comment]
I recently tried a `pip install git+https://github.com/Feneric/doxypypy`
but the same error popped up, so if there is a fix it's not in the current source I think.

Looking here though I'm not sure if this obsolete's doxypypy unless doxypypy is still doing other things to fix up the python doxygen output https://github.com/Feneric/doxypypy/issues/88

**Comment by JoaoPedroMDP** [comment]
I had this same issue here. Python 3.10.
`TypeError: NodeVisitor.visit_Constant() got an unexpected keyword argument 'containingNodes'`
